shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -436,6 +480,8 @@ public Options getOptions() {
     outputFileOpt = new Option("o", "output", true, "local file to write the scan output to");
     namespaceOpt = new Option(ShellOptions.namespaceOption, "namespace", true,
         "namespace to display/set/delete properties for");
+    resourceGroupOpt = new Option(ShellOptions.resourceGroupOption, "resourceGroup", true,
```

Comment:
Not that the `rg` option is only for setting or deleting a resource group property. The ConfigCommand displays all resource group values when listing properties. The command currently does not display a single resource groups configuration based on the `rg` property due to the system configuration merged view issue noted in a different comment. It may be possible though, to use the `rg` property to determine which host to connect to (instead of random).